keyboards/westfoxtrot/cypher/rev1/rev1.c

```diff
@@ -13,7 +13,21 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+#include "rev1.h"
 
-#pragma once
-
-// place overrides here
+bool led_update_kb(led_t led_state) {
+    bool res = led_update_user(led_state);
+    if(res) {
+        // writePin sets the pin high for 1 and low for 0.
+        // In this example the pins are inverted, setting
+        // it low/0 turns it on, and high/1 turns the LED off.
+        // This behavior depends on whether the LED is between the pin
+        // and VCC or the pin and GND.
+        writePin(F4, !led_state.num_lock);
+        writePin(F1, !led_state.caps_lock);
+        writePin(F5, !led_state.scroll_lock);
```

Comment:
Do your indicator LEDs work correctly from this branch? It's not clear to me from the other conversation.

Reply:
they now do.